joomla/joomla-cms
# J4.x Fix Pagination Float and Wrap on mobile Cassiopeia

I actually wanted to create a PR for issue https://github.com/joomla/joomla-cms/issues/32674 but it's wrongly assigned to frontend while the Screenshot in the Issue is the Backend. However the Pagination in Frontend was broken on mobile too, so I fixed it with flex-wrap class in the pagination and I adjusted the Pages Information to only float-end from mid-screen size.

### Testing Instructions
Create some articles to have a large Pagination list or fake it with the browser inspector.
On Mobile the pagination should not break the layout.
- Check Blog and Category List Layout
- Check Com Contact Layout
- Archive Layout

### Actual result BEFORE applying this Pull Request
<img width="420" alt="grafik" src="https://user-images.githubusercontent.com/828371/166144650-15aed255-8be5-4675-b811-716046838126.png">
<img width="513" alt="grafik" src="https://user-images.githubusercontent.com/828371/166144660-318e5d35-bde2-4b41-9e97-e7bc761facd2.png">

Don't mind the wrong pagination numbers as I faked the page Items instead of creating more articles)


### Expected result AFTER applying this Pull Request
<img width="457" alt="grafik" src="https://user-images.githubusercontent.com/828371/166144723-c6a8cfba-ee49-4c7b-b32b-f2b06d4b9f19.png">


### Documentation Changes Required
No
